src/Interpolation.elm

```diff
@@ -153,16 +153,6 @@ tuple ia ib ( fromA, fromB ) ( toA, toB ) =
 
 
 
--- Does this even make any sense?
-
-
-andThen : (a -> Interpolator b) -> Interpolator a -> Interpolator b
-andThen fn interpolator =
-    \param ->
-        fn (interpolator param) param
-
-
-
```

Comment:
I though it was some forgotten floating comment :).
I think we can clarify the comment a bit.

Reply:
Can you suggest a more fitting comment?
Should andThen be exposed?

Reply:
Something like:

```
-- We can also provide an `andThen` function for our Interpolator:
--
--     andThen : (a -> Interpolator b) -> Interpolator a -> Interpolator b
--     andThen fn interpolator =
--           \param ->
--                 fn (interpolator param) param
--
--   However, we have not actually come up with a scenario where this would make sense.
```